internal/provider/egress_private_endpoint_resource_test.go

```diff
@@ -0,0 +1,280 @@
+/*
+ Copyright 2025 The Cockroach Authors
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package provider
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/cockroachdb/cockroach-cloud-sdk-go/v6/pkg/client"
+	mock_client "github.com/cockroachdb/terraform-provider-cockroach/mock"
+	"github.com/golang/mock/gomock"
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestAccEgressPrivateEndpointResource(t *testing.T) {
```

Comment:
does the acceptance test rely on cloud resources?  I didn't see anything specific so I was wondering how this would work.

Reply:
nice.  Thanks for the explanation.